### ISSUE TITLE
Project Transfer

### ISSUE THREAD

**Issue by MartinThoma** [idin/pdf#2]
Hello Idin,

I'm currently doing a lot with various PDF packages (pikepdf, pdfminer.six, pymupdf) and noticed that there are quite a lot of Python packages that are not maintained any longer (PyPDF, PyPDF2, PyPDF3, PyPDF4, pdfminer, ...)

I would like to create an interface for interacting with PDF which uses a (potentially exchangeable) backend, similar to the machine learning project Keras which can switch the backend between Tensorflow and Theano.

Just "pdf" is a pretty awesome name. Would you be open to transferring the PyPI "pdf" project to me?

Best regards,
Martin

**Comment by MartinThoma** [comment]
@idin I've just became the owner of PyPDF2 (on PyPI and on Github) and I created the py-pdf github organization: https://github.com/py-pdf/PyPDF2

Do you think the `pdf` package is used at all? (looking at the [download stats](https://pypistats.org/packages/pdf) for such a popular name, I don't think so)

I would like to re-purpose the `pdf` package on PyPI to have one well-designed package for handling PDF files with Python. What do you think about it?

**Comment by MartinThoma** [comment]
I've also written you an email on 3rd of April (2022) in case you want to discuss this in private.

**Comment by ariebovenberg** [comment]
@MartinThoma in case you didn't know, you can request project transfer through the official channels here: https://github.com/pypa/pypi-support/labels/PEP%20541

**Comment by MartinThoma** [comment]
Thank you for the reminder. I'm aware of that possibility, but before I go this route I wanted to try it by talking to the current "owner" of that package name.

It seems like this is not possible. It also seems as if the package is not used (looking at the stars / watchers on Github).

I became a project member of `pypdf` as well and I'm thinking about simply moving `pypdf2` to `pypdf`.

Regarding the `pdf` package, I'm uncertain if I should attempt to get it and what should be behind that package. What do you think @ariebovenberg ?

**Comment by MartinThoma** [comment]
I want to avoid to move `pypdf2` to `pypdf` and then to `pdf`.

As `pypdf2` is a pure-python package and now likely has a stable or growing community / dev setup / user base, it might be worth using this package name. But I'm for sure biased :smile:
